### PR TITLE
136 credential definition detailed overview popup page

### PR DESCRIPTION
**Pull request by valentinBraeutigam**
We've implemented the credential definition overview page as a pop up window. Tests will be added in the future.

RM: @valentinBraeutigam

### CODE REVIEW

File: src/digitalIdentity-frontend/src/app/components/credDefDetail-table-pop-up/credDefDetail-table-pop-up.component.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<h1>Details to credential definition {{credDef.alias}}</h1>
```
Comment (Idontker) [inline src/digitalIdentity-frontend/src/app/components/credDefDetail-table-pop-up/credDefDetail-table-pop-up.component.html:1]: use mat-dialog-title, mat-dialog-content, mat-dialog-actions within the dialog template to organize the dialog.

File: src/digitalIdentity-frontend/src/app/pages/credential/credential-overview-page/credDef-overview-page.component.ts [hunk 2]
```diff
@@ -33,11 +34,19 @@ export class CredDefOverviewPageComponent implements OnInit {
     if (isDevMode()) {
       console.log("Edit")
     }
+    console.log("Test")
```
Comment (Idontker) [inline src/digitalIdentity-frontend/src/app/pages/credential/credential-overview-page/credDef-overview-page.component.ts:37]: should be deleted

**Review by Idontker — CHANGES_REQUESTED** [review]
- [x] More Details binded to edit button
- [x] missing acceptance criteria:
  - [x] when was the credential assigned to a DI
  - [ ] when was the credential revoked for a DI
- [ ] run prettier
- [ ] tests are missing
- [ ] text within table not readable 
image

File: src/digitalIdentity-frontend/src/app/components/credDefDetail-table-pop-up/credDefDetail-table-pop-up.component.html (hunk 1 above)
Comment (valentinBraeutigam) [reply src/digitalIdentity-frontend/src/app/components/credDefDetail-table-pop-up/credDefDetail-table-pop-up.component.html:1]: Done

File: src/digitalIdentity-frontend/src/app/pages/credential/credential-overview-page/credDef-overview-page.component.ts (hunk 2 above)
Comment (valentinBraeutigam) [reply src/digitalIdentity-frontend/src/app/pages/credential/credential-overview-page/credDef-overview-page.component.ts:37]: Done :)

**Pull request merged**